### PR TITLE
List versions

### PR DESCRIPTION
**Pull request by pavloo**
Implementation of `deploy:versions` command.

### CODE REVIEW

**Comment by pavloo** [comment]
@achambers Is not ready for merge

File: tests/unit/tasks/deploy-versions-test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  it('resolves with versions', function() {
+    MockAdapter.prototype.listVersions = resolved.bind(this, [{ key: 'aaa' }, {key: 'bbb' }]);
+    options.registry = new MockRegistry();
+
+    subject = new DeployVersionsTask(options);
+
+    return subject.run({environment: 'development'})
+    .then(function(outputStr) {
+      assert.equal(mockUI.output, 'aaa\nbbb');
```
Comment (pavloo) [inline tests/unit/tasks/deploy-versions-test.js]: @achambers Are u ok with the format of output?

File: tests/unit/commands/deploy-index-test.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ var Promise      = require('ember-cli/lib/ext/promise');
 var MockRegistry = require('../../helpers/mock-registry');
 
 describe('deploy:index command', function() {
-  var subject
-  var tasks;
```
Comment (achambers) [inline tests/unit/commands/deploy-index-test.js:10]: Can you please keep this as is?  ie, a `var` on each line.

File: tests/unit/tasks/deploy-versions-test.js (hunk 1 above)
Comment (achambers) [inline tests/unit/tasks/deploy-versions-test.js]: We have an issue to improve the command line messaging across the board (#53).

However if you could maybe add a line above the version keys saying something to the effect of:

'The previous x versions uploaded:'

Something like that anyway

**Comment by achambers** [comment]
Could you also please re-raise this PR against the `next-release-0.0.6` branch?

**Pull request closed without merging**